idaes/power_generation/unit_models/soec_design.py

```diff
@@ -0,0 +1,686 @@
+            destination=self.o2_seperator.inlet,
+        )
+        self.strm3 = Arc(
+            doc="Oxygen seperation to hydrogen side outlet translator",
+            source=self.o2_seperator.h2_strm,
+            destination=self.h2_outlet_translator.inlet,
+        )
+        self.strm4 = Arc(
+            doc="Oxygen seperation to O2 to sweep translator",
+            source=self.o2_seperator.o2_strm,
+            destination=self.o2_translator.inlet,
+        )
+        self.strm5 = Arc(
+            doc="Oxygen translator to O2/sweep mixer",
+            source=self.o2_translator.outlet,
+            destination=self.o2_mixer.o2_strm,
+        )
+        self.strm6 = Arc(
+            doc="O2/sweep mixer to sweep heater",
+            source=self.o2_mixer.outlet,
+            destination=self.sweep_heater.inlet,
+        )
+        # Probably don't need this and may need to call it again for dynamic
+        # models, but this will make it easy for a user to have a flowsheet
+        # with one steady state model in it.
+        pyo.TransformationFactory("network.expand_arcs").apply_to(self)
+
+    def _add_variables(self):
+        """Add variables for unit model level constraints"""
+        self.current = pyo.Var(
+            self.flowsheet().time,
+            initialize=0,
+            units=pyo.units.ampere,
+            doc="SOEC module electric currrent",
+        )
+        self.water_utilization = pyo.Var(
+            self.flowsheet().time,
+            units=pyo.units.dimensionless,
+            doc="Fraction of water used in electrolysis reaction"
+        )
+        self.cell_potential = pyo.Var(
+            self.flowsheet().time,
+            initialize=1.3,
+            units=pyo.units.volts,
+            doc="Electric potential for a single cell"
+        )
+        self.heat = pyo.Var(
+            self.flowsheet().time,
+            units=pyo.units.W,
+            initialize=0,
+            doc="Heat transfered to the module"
+        )
+        self.hydrogen_side_outlet_temperature = pyo.Reference(
+            self.electrolysis_reactor.control_volume.properties_out[:].temperature
+        )
+        self.oxygen_side_outlet_temperature = pyo.Reference(
+            self.sweep_heater.control_volume.properties_out[:].temperature
+        )
+
+    def _add_constraints(self):
+        """Add unit model constraints"""
+        @self.Constraint(self.flowsheet().time)
+        def water_utilization_eqn(b, t):
+            return (
+                b.electrolysis_reactor.control_volume.properties_out[t].flow_mol *
+                b.electrolysis_reactor.control_volume.properties_out[
+                    t
+                ].mole_frac_comp["H2O"] ==
+                b.electrolysis_reactor.control_volume.properties_in[t].flow_mol *
+                b.electrolysis_reactor.control_volume.properties_in[
+                    t
+                ].mole_frac_comp["H2O"] *
+                (1.0 - b.water_utilization[t])
```

Comment:
Minor, but why not just use `flow_mol_comp` here?

Reply:
No reason.  Good call that's a bit cleaner.